src/libraries/Common/tests/System/Security/Cryptography/AlgorithmImplementations/AES/AesCipherTests.cs

```diff
@@ -682,95 +682,107 @@ public static void VerifyKnownTransform_CFB8_128_PKCS7_4()
                 feedbackSize: 8);
         }
 
-        [Fact]
-        public static void VerifyKnownTransform_CFB8_128_NoPadding_0_Extended()
+        [Theory]
+        [InlineData(PaddingMode.None)]
+        [InlineData(PaddingMode.Zeros)]
+        public static void VerifyKnownTransform_CFB8_128_NoOrZeroPadding_0_Extended(PaddingMode paddingMode)
```

Comment:
Since `Zero` padding does nothing when the input is aligned correctly, and for CFB8 means "any length is aligned correctly", there is no meaningful difference between `None` and `Zeros` when using CFB8, so we can re-use that existing tests.